packages/tests/src/nwaku.ts

```diff
@@ -440,7 +451,7 @@ export function defaultArgs(): Args {
   return {
     listenAddress: "127.0.0.1",
     nat: "none",
-    relay: true,
+    // relay: true,
```

Comment:
Yes I agree, just remove this default (you may have to change some tests).

```suggestion
```

Reply:
addressed this here: https://github.com/waku-org/js-waku/pull/1044